### ISSUE TITLE
Download button is not accessible using the keyboard: A11y_VS Code extension for API Center_Open API Documentation_Download_Keyboard.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/vscode-azureapicenter#93]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG2.1.1; #GH_VSCodeextensionforAPICenter_Win32_Apr2024; #DesktopApp; #VS Code Extension for API Center; #Keyboard; #A11yMAS; #External; #External:Swagger; #Thirdparty; eDAD:3P:Bug222940;

### Environment Details:
Application Name: VS Code Extension for API Center
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)

### Repro Steps:
1. Create an API center in azure portal.
2. Open Visual studio Code.
3. Tab till API center extension in the left side panel and press enter.
4. Tab till definition under the Api name which you have created, and press enter.
5. The name which you have provided to the definition will appear for ex-"openapi4", right click on it.
6. Go to "Open API Documentation" using the down arrow and press enter.
7.  Demo conference API screen will appear.
8. Tab till download button.
9. Verify whether the download button is accessible using the keyboard or not.

### Actual Result:
Download button is not accessible using the keyboard when we are navigating using the tab key.

### Expected Result:
Download button should be accessible using the keyboard keys.

### User Impact:
keyboard users will not be able to access the button.

### Attachment:
![Bug_93](https://github.com/microsoft/vscode-azureapicenter/assets/124244478/9069d295-a93d-4732-b3e8-40c593d89ab2)

**Comment by AnKushSingh05** [comment]
@formulahendry Can you please provide us the standalone sample app or Code pen link to route this bug to external team.

**Comment by AnKushSingh05** [comment]
IRT Bug logged: https://dev.azure.com/CSS-IRT/CSS%20IRT/_workitems/edit/222940

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment.
Version: 1.88.1 (user setup)
Commit: e170252f762678dec6ca2cc69aba1570769a5d39
Date: 2024-04-10T17:41:02.734Z
Issue is fixed. Hence closing the bug

https://github.com/microsoft/vscode-azureapicenter/assets/124244478/6dc733d8-a789-41e7-8e4d-c90f28565a18
